include/pcmki/pcmki_result_code.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PCMK__PCMKI_PCMKI_RESULT_CODE__H
```

Comment:
Remember the copyright boilerplate at the top of the file.

Reply:
Done, thanks. Like I said in #2836, the existing one I used as a template didn't have the copyright. Fixed that in a new commit in #2836.